docs/examples/driver_examples/Qcodes example with DynaCool PPMS.ipynb

```diff
@@ -976,11 +976,11 @@
     "\n",
     "The field has **four** related parameters:\n",
     "\n",
-    "- `field_measured`: (read-only) the field strength right now\n",
-    "- `field_target`: the target field that the `ramp` method will ramp to when called. Setting this parameter does **not** trigger a ramp\n",
-    "- `field_rate`: the field ramp rate. NB: setting this parameter **will** trigger a ramp\n",
-    "- `field_approach`: the approach the system should use to ramp. NB: setting this parameter **will** trigger a ramp\n",
-    "- `field_ramp`: this is a convenience parameter that sets the target and then triggers a blocking ramp.\n",
+    "- `field_measured`: The (read-only) field strength right now.\n",
+    "- `field_target`: The target field that the `ramp` method will ramp to when called. Setting this parameter does **not** trigger a ramp\n",
+    "- `field_rate`: The **cached** field ramp rate with initial value of `0`.\n",
+    "- `field_approach`: The **cached** approach that the system should use to ramp. By default it is set to `linear`.\n",
```

Comment:
what does "cached" mean here exactly? the fact that this parameter in in-memory only and does not correspond 1-1 to any "setting" of the instrument?

Reply:
Yes exactly, your understanding spot on. Do you feel it is not clear?

Reply:
yeah... i wouldn't want it to be confused with the "internal cache that each qcodes parameter has"... but i also don't know any other good word to use here iso "cached" - perhaps i'd just remove that word - do users usually care much about these details?

Reply:
I believe a common user would not be interested to know how. I just wanted to be transparent but I now also agree with you. I will remove the word.